### PR TITLE
Generate (but don't publish) the Live Stream API

### PR DESCRIPTION
**Pull request by jskeet**
(The API is not yet accessible.)

### CODE REVIEW

File: apis/Google.Cloud.Video.LiveStream.V1/.OwlBot.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+# Copyright 2021 Google LLC
```
Comment (amanda-tarafa) [inline apis/Google.Cloud.Video.LiveStream.V1/.OwlBot.yaml:2]: nit: This one is still beeing generated with 2021

**Review by amanda-tarafa — APPROVED** [review]
LGTM

File: apis/Google.Cloud.Video.LiveStream.V1/.OwlBot.yaml (hunk 1 above)
Comment (jskeet) [reply apis/Google.Cloud.Video.LiveStream.V1/.OwlBot.yaml:2]: Ooh, interesting. Thanks, will fix.

Comment (jskeet) [reply apis/Google.Cloud.Video.LiveStream.V1/.OwlBot.yaml:2]: (In a separate PR though, as I suspect it's the generator.)

**Pull request merged**